pkg/proxy/ipvs/README.md

```diff
@@ -27,27 +27,17 @@ modprobe nf_conntrack_ipv4
 
 After that, use `lsmod | grep ip_vs` to make sure kernel modules are loaded.
 
-##### Run kube-proxy in ipvs mode
+#### Run kube-proxy in ipvs mode
 
-First, [run cluster locally](https://github.com/kubernetes/community/blob/master/contributors/devel/running-locally.md). 
+#### Local UP Cluster
 
-By default kube-proxy will run in iptables mode, with configuration file `/tmp/kube-proxy.yaml`. so we need to change the
-configuration file and restart it. Here is a yaml file for reference.
+Kube-proxy will run in iptables mode by default in a [local-up cluster](https://github.com/kubernetes/community/blob/master/contributors/devel/running-locally.md). 
 
-```yaml
-apiVersion: componentconfig/v1alpha1
```

Comment:
Does this file not officially work?

Reply:
Edit this file is a temporary hack. There is no need to hack this file thanks to the work of #54742.